metricbeat/module/system/process/helper.go

```diff
@@ -20,19 +21,20 @@ import (
 type ProcsMap map[int]*Process
 
 type Process struct {
-	Pid      int    `json:"pid"`
-	Ppid     int    `json:"ppid"`
-	Pgid     int    `json:"pgid"`
-	Name     string `json:"name"`
-	Username string `json:"username"`
-	State    string `json:"state"`
-	CmdLine  string `json:"cmdline"`
-	Cwd      string `json:"cwd"`
-	Mem      sigar.ProcMem
-	Cpu      sigar.ProcTime
-	Ctime    time.Time
-	FD       sigar.ProcFDUsage
-	Env      common.MapStr
+	Pid         int    `json:"pid"`
+	Ppid        int    `json:"ppid"`
+	Pgid        int    `json:"pgid"`
+	Name        string `json:"name"`
+	Username    string `json:"username"`
+	State       string `json:"state"`
+	CmdLine     string `json:"cmdline"`
+	Cwd         string `json:"cwd"`
+	Mem         sigar.ProcMem
+	Cpu         sigar.ProcTime
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
struct field Cpu should be CPU